plugins/module_utils/_api_data.py

```diff
@@ -4423,6 +4423,7 @@ def join_path(path):
             fully_understood=True,
```

Comment:
Maybe this way would be better though:
```suggestion
            fully_understood=True,
            versioned_fields=[
                ([('7.14', '>=')], 'show-at-cli-login', KeyInfo(default=False)),
            ],
```